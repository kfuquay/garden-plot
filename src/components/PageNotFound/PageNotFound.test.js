import React from "react";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";
import PageNotFound from "./PageNotFound";

describe(`Page Not Found component`, () => {
  it("renders without crashing", () => {
    const wrapper = shallow(<PageNotFound />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
