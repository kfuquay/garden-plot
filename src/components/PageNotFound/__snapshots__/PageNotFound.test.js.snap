// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Page Not Found component renders without crashing 1`] = `
<p>
  Page Not Found
</p>
`;
